pkg/cli/set/scheme.go

```diff
@@ -0,0 +1,27 @@
+package set
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+
+	"github.com/openshift/api"
+	"github.com/openshift/api/route"
+
+	schemehelper "github.com/openshift/oc/pkg/helpers/scheme"
+)
+
+var setCustomScheme = runtime.NewScheme()
```

Comment:
this name is still unfortunately ambiguous,  can we for example name it `setCmdScheme` ?

Reply:
Agree and `setCmdScheme` sounds good to me